helm-chart/Chart.yaml

```diff
@@ -22,4 +22,4 @@ name: kubeshark
 sources:
   - https://github.com/kubeshark/kubeshark/tree/master/helm-chart
 type: application
-version: "40.0"
+version: "0.1.0"
```

Comment:
This version should match the CLI version `kubeshark version` which is `40.0` at this point.

I guess it's related to https://github.com/kubeshark/kubeshark/issues/1348

Please see the note in [here](https://github.com/kubeshark/kubeshark/wiki/CHANGELOG) to understand how we ended up with something other than SemVer. So, the repository version scheme should change for this.